### PR TITLE
feat(Engines): add Camunda 8.2 profile

### LINKED ISSUE #3284: Add Camunda 8.2 profile
### What should we do?

The [first alpha](https://github.com/camunda/zeebe/releases) of Camunda 8.2 is out. As a user I want to be able to use it in the Modeler.

* [ ] Be able to select 8.2 as a diagram target (try out new features)
* [ ] Keep 8.1 (stable) as a default for now

### Why should we do it?

Make it possible for me as a user to use 8.2 features in my diagrams (https://github.com/camunda/camunda-modeler/issues/3282).

### PR DESCRIPTION
**Pull request by marstamm**
I decided to stick with "Camunda 8.2" as target label (instead of "8.2-alpha") to keep the naming consistent with the lint warnings. 

Recording 2022-11-09 at 10 53 18

closes #3284

### CODE REVIEW

File: client/src/app/TabsProvider.js [hunk 1]
```diff
@@ -698,7 +698,7 @@ function replaceVersions(contents) {
 
   const latestCloudVersion = ENGINE_PROFILES.find(
     p => p.executionPlatform === ENGINES.CLOUD
-  ).executionPlatformVersions[0];
+  ).latestStable;
```
Comment (nikku) [inline client/src/app/TabsProvider.js]: I propose to apply this new feature consistently, i.e. also for C7 (https://github.com/camunda/camunda-modeler/pull/3286/files#diff-a8ee53f63c47900983b24616f87f9b04023e1b48be19fd3ee726d0138276b811L697):

Comment (nikku) [reply client/src/app/TabsProvider.js]: Cf. https://github.com/camunda/camunda-modeler/pull/3286/commits/959d1d693284197568a902a931ef16b680e0a226 and https://github.com/camunda/camunda-modeler/pull/3286/commits/7cc30096f26b843cfeba6cda9a8f36322bc6b5c5.

Comment (marstamm) [reply client/src/app/TabsProvider.js]: makes sense to me. Should we then also support pre-release Camunda 7 versions? 7.19 has an alpha out already (cf. https://docs.camunda.org/enterprise/download/,  [94dccc4](https://github.com/camunda/camunda-modeler/pull/3286/commits/94dccc4b103bb25cf47118caaf33aa0354414af2))

Comment (nikku) [reply client/src/app/TabsProvider.js]: Yes in this case! :)

**Review by nikku — APPROVED** [review]

File: client/src/app/TabsProvider.js (hunk 1 above)
Comment (marstamm) [reply client/src/app/TabsProvider.js]: 👍 

I'll squash the commits real quick

**Pull request merged**